### PR TITLE
Added BSB / BetterPortal technology

### PR DESCRIPTION
**Pull request by mrinc**
BSB runs the backend of BetterPortal (and other systems in the future) - so tying it in a single merge.  

Examples:  
https://go.betterportal.cloud/  
https://forms-za.betterportal.cloud/  
https://forms.bettercorp.co.za/  
https://ninja.bettercorp.co.za/

### CODE REVIEW

File: src/technologies/b.json [hunk 1]
```diff
@@ -493,6 +493,43 @@
...
+    "website": "https://bsbcode.dev/"
+  },
+  "BetterPortal": {
+    "cats": [
+      9,
+      46,
+      53,
+      58,
+      66,
+      88
```
Comment (AliasIO) [inline src/technologies/b.json:513]: Please choose a single category if possible, or two at most.

File: src/technologies/b.json [hunk 2]
```diff
@@ -493,6 +493,43 @@
...
+    "description": "BetterPortal ",
+    "icon": "BetterPortal.png",
+    "implies": [
+      "Better-Service-Base"
+    ],
+    "meta": {
+      "generator": "BetterPortal"
+    },
+    "js": {
+      "bsb": ""
```
Comment (AliasIO) [inline src/technologies/b.json:524]: Is there another variable we can use? Short variables like these often appear in minified code and can lead to false detections.

**Comment by AliasIO** [comment]
Thanks! Please note the comments I left and reduce the PNGs to 32x32 if you can.

File: src/technologies/b.json (hunk 2 above)
Comment (mrinc) [reply src/technologies/b.json:524]: Okay, changed the platform a bit to work better here.

Would `window.bsb.betterportal` -> thus `bsb.betterportal` be better suited in this context?

Comment (AliasIO) [reply src/technologies/b.json:524]: Yes, `bsb.betterportal` works. 👌

File: src/technologies/b.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/b.json:513]: > Thanks! Please note the comments I left and reduce the PNGs to 32x32 if you can.

BetterPortal
user-images.githubusercontent.com/3456008/199805649-947caa7e-6b2f-4973-bc12-18d8fec3dfc4.png)

**Comment by mrinc** [comment]
@AliasIO Sorry crazy few weeks. 
Will do the updates tomorrow and push an updated request :) 

Thanks

**Pull request closed without merging**